packages/synapse-react-client/src/mocks/msw/handlers/featureFlagHandlers.ts

```diff
@@ -1,5 +1,4 @@
 import { rest } from 'msw'
-import { server } from '../server'
```

Comment:
`server.ts` imports `handlers.ts` which imports this file, so we cannot import from `server.ts` here.